src/socketEvent/game-event/game.enum.ts

```
@@ -31,3 +37,24 @@ export interface LeaveGame {
   sessionId: string;
 }
+
+export interface InviteGame {
+  senderId: string;
+  receiverId: string;
+  gameId: string;
+  note: string;
+}
+
+
+export interface AcceptInvite {
```

ESLint

[error] Delete `⏎`

(prettier/prettier)